packages/bits/src/pipes/get-step.pipe.ts

```diff
@@ -0,0 +1,26 @@
+import { Pipe, PipeTransform, QueryList } from "@angular/core";
+import { WizardStepV2Component } from "../lib/wizard-v2/wizard-step/wizard-step.component";
+
+
+/**
+ * GetStep pipe is used to get the wizard step by index
+ *
+ * __Parameters :__
+ *
+ *   index - the index of step
+ *
+ * __Usage :__
+ *   steps | getStep:index
+ *
+ * __Example :__
+ *   <code>{{ steps | getStep:3 }}</code>
+ *
+ */
+@Pipe({
+    name: "getStep",
+})
+export class GetStep implements PipeTransform {
```

Comment:
This pipe is very specific to the wizard. Do we really want to export it to our consumers? Maybe, we may keep with the wizard

Reply:
Moved, thanks